packages/apollo-server-types/src/index.ts

```diff
@@ -184,14 +185,6 @@ export type GraphQLExecutionResult = {
   extensions?: Record<string, any>;
 };
 
-export type Logger = {
```

Comment:
This seems like a backwards-incompatible change that we shouldn't do on `main`. Instead we should export what we imported?

Reply:
Yeah, good catch. [f3dbe44](https://github.com/apollographql/apollo-server/pull/6299/commits/f3dbe44efaa65653e7917c0c6ecd1438109523f5)